src/utils/VectorFactory.ts

```diff
@@ -16,6 +16,9 @@
  */
 export function wktPolygonToSvg(wkt: any , boundingRect?: any) {
```

Comment:
wkt should be a string (I'm not sure what `wkt.wkt` would be, but that looks odd), and boundingRect should be a rect object with x, y, width, and height.

Reply:
This was taken from the old website. We did not add rectangles, matrices or other types yet, and I'd rather not do that. I don't know what `wkt.wkt` is, but if it's there - it probably is something.

Reply:
nice...  I remember some of this code, and think when you pass in a rect (usually the bounding box of the artefact in the master image coordinate system), it will transform the polygon into that coordinate system.  We very well may not use that approach in new code (it was used in the "virtual scroll" panel).